### ISSUE TITLE
generate hosts entry from k8s service

### ISSUE THREAD

**Issue by kyr1eeee** [kubenetworks/kubevpn#120]
This is my situation：

The k8s service spec in my cluster is like below:
```yaml
spec:
  clusterIP: 12.12.227.122
  externalTrafficPolicy: Cluster
  ports:
  - nodePort: 30234
    port: 8004
    protocol: TCP
    targetPort: 8004
  selector:
    dev: kk
  sessionAffinity: None
  type: NodePort
```
As you can see, spec has no ClusterIPs field. But kubevpn only read ClusterIPs for clusterIp parsing:
```golang
func (c *Config) generateHostsEntry(list []v12.Service, hosts []Entry) []Entry {
	const ServiceKubernetes = "kubernetes"
	var entryList = sets.New[Entry]().Insert(c.Hosts...).Insert(hosts...).UnsortedList()

	// get all service ip
	for _, item := range list {
		if strings.EqualFold(item.Name, ServiceKubernetes) {
			continue
		}
                // only read Spec.ClusterIPs filed
		ipList := sets.New[string](item.Spec.ClusterIPs...).Insert(item.Spec.ExternalIPs...).UnsortedList()
		domainList := sets.New[string](item.Name).Insert(item.Spec.ExternalName).UnsortedList()
		for _, ip := range ipList {
			for _, domain := range domainList {
				if net.ParseIP(ip) == nil || domain == "" {
					continue
				}
				entryList = append(entryList, Entry{IP: ip, Domain: domain})
			}
		}
	}
```
So i can't reach k8s svc with service name in my local pc.
Could the svc.Spec.CLusterIP field be considered when generateHostsEntry?

**Comment by wencaiwulue** [comment]
Yes, you are right, i will fix this issue as soon as possible

**Comment by wencaiwulue** [comment]
I redo release v2.2.1, fixed this issue, you can download latest v2.2.1 to have a try. [download link](https://github.com/kubenetworks/kubevpn/releases/tag/v2.2.1). 
Now: it support two type of service on Windows:
- visit k8s service via service name. eg: curl my-service:8080
- visit k8s service via full dns name. eg: curl my-service.default.svc.cluster.local:8080
